src/GeniusYield/GYConfig.hs

```diff
@@ -119,9 +119,10 @@ In JSON format, this essentially corresponds to:
 = { coreProvider: GYCoreProviderInfo, networkId: NetworkId, logging: [GYLogScribeConfig], utxoCacheEnable: boolean }
 -}
 data GYCoreConfig = GYCoreConfig
-  { cfgCoreProvider :: !GYCoreProviderInfo
-  , cfgNetworkId    :: !GYNetworkId
-  , cfgLogging      :: ![GYLogScribeConfig]
+  { cfgCoreProvider  :: !GYCoreProviderInfo
+  , cfgNetworkId     :: !GYNetworkId
+  , cfgLogging       :: ![GYLogScribeConfig]
+  , cfgMaestroTiming :: !Bool
```

Comment:
This has nothing to do with Maestro and is to be applicable for all providers.

Reply:
OK, even simpler.

Note that I have kept the commented lines 209-215.  Without these and intermediate variables `(gyQueryUTxO', gySlotActions')`, code could be simplified.